app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -603,11 +566,32 @@ class BrowserFragment : LocaleAwareFragment(),
             }
 
             fragment.permissionHandler.tryAction(
-                    fragment,
-                    Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                    ACTION_DOWNLOAD,
-                    download
-                    )
+                fragment,
+                Manifest.permission.WRITE_EXTERNAL_STORAGE,
+                ACTION_DOWNLOAD,
+                download
+            )
         }
     }
 }
+
+private fun HitResult.toHitTarget(): PrivateWebContextMenu.HitTarget {
+    val isLink: Boolean
+    val linkURL: String
+    var isImage = false
```

Comment:
isImage is always false here.

Reply:
isImage will be set to true below at line:587 only if the AC HitTarget is HitResult.IMAGE_SRC